internal/prometheus/collector.go

```diff
@@ -0,0 +1,257 @@
+package prometheus
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/vinted/graphql-exporter/internal/config"
+	"github.com/vinted/graphql-exporter/internal/graphql"
+)
+
+var latencyHistogramBuckets = []float64{.1, .25, .5, 1, 2.5, 5, 10, 15, 20, 30, 40, 50, 60, 90, 150, 210, 270, 330, 390, 450, 500, 600, 1200, 1800, 2700, 3600}
```

Comment:
Do we really need that much buckets for latency? you set a 60s timeout limit so everything > 60 seems unlikely, can we use the [default value](https://github.com/prometheus/client_golang/blob/1ba60c7d585210b1df898b4077ed8cf45b4bf0d2/prometheus/histogram.go#L54-L59) instead?